text.txt

```diff
@@ -1,8 +1,14 @@
 Vitajte v Progressbare!
 
-Progressbar je občianske združenie a priestor, v ktorom sa stretávajú hackeri, geekovia, umelci, nadšenci do elektroniky a dokonca aj psychológovia.
-Máme tu labák, prednáškovú miestnosť, coworking space a kuchynku. Ak by ste potrebovali ísť na WC, nájdete ho vzadu za kuchynkou.
+Progressbar je priestor, v ktorom sa stretávajú ľudia rôznych povolaní, záujmov a kultúr. 
+Hackeri, geekovia, umelci, nadšenci do elektroniky, psychológovia a mnoho iných.
+Máme tu laboratórium, prednáškovú miestnosť, coworking space a kuchynku. 
+Toaleta sa nachádza vzadu za kuchynkou.
 
-V kuchynke funguje samoobslužný systém. Môžete si zobrať na čo máte chuť a prispieť poslaním Bitcoinov alebo vhodením Eur do pokladničky.
+V kuchynke funguje samoobslužný systém. 
+Môžete si zobrať na čo máte chuť a prispieť poslaním Bitcoinov alebo vhodením Eur do pokladničky.
 
-Ako občianske združenie fungujeme z členského a dobrovoľných príspevkov návštevníkov. Poteší nás každý príspevok a ak sa budete chcieť stač členom, môžete sa zaregistrovať na našej stránke. Všetci sú u nás vítaní!
+Ako občianske združenie fungujeme z členského a dobrovoľných príspevkov návštevníkov. 
+Poteší nás každý príspevok a nový členovia sú vždy vítaní!
+
+Viac informácií nájdete na https://www.progressbar.sk/pridaj-sa
```

Comment:
Ono debata url s www alebo bez je roky a obe maju svoje vyhody a nevyhody. 
Napriklad tu su niektore popisane http://www.greenhills.co.uk/2011/06/12/www-or-bare-domain.html Kazdopadne je to len minor zalezitost v ramci textu a ak myslis, ze to v texte bude lepsie bez www tak PR updatnem.